azure-pipelines.yml

```diff
@@ -45,84 +46,83 @@ stages:
       - name: vs
       - name: broadcom
       - name: mellanox
-
-- stage: Test
```

Comment:
If you want to disable the Test stage, please use.
condition: false

@qiluo-msft , do you think we can only enable the build without Test? It should be better than without any check in 2001911.

Reply:
Agree. At lease we have some build checker, better than nothing.